deploy/clowdapp.yaml

```diff
@@ -1012,6 +1056,18 @@ parameters:
 - name: MEMORY_LIMIT_GROUPER
   description: Maxmimum memory limit for pod
   value: "512Mi"
+- name: CPU_LIMIT_CLUSTER
+  description: Maximum CPU limit for pod
+  value: #...
```

Comment:
Lets use the same limit / request & cpu / memory values as in exploit sync, the small values that it is using should enough for us